lib/ansible/modules/cloud/vmware/vmware_guest_snapshot.py

```diff
@@ -60,6 +60,7 @@
    folder:
         description:
             - Define instance folder location.
+        required: True
```

Comment:
this doesn't look correct
`folder=dict(required=False, type='str', default='/vm'),`

I think it should actually be
`default: /vm` on this line